index.js

```diff
@@ -43,6 +44,9 @@ client.once('ready', async () => {
 	courseRequest.getCourseRequests(mainChannel);
 	// Executes the function getCourseRequests every 1 hour (=3,600,000 millisecs).
 	setInterval(() => courseRequest.getCourseRequests(mainChannel), 3600000);
```

Comment:
Both of these should be 60000 ms (1 minute). It is set to a minute in the actual server but not synced with github.